common/changes/lju-personaSizes_2017-03-23-19-16.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Added a 28px by 28px persona control.",
```

Comment:
Minor formatting request to keep the release notes consistent: "Persona: Added a 28px size"